index.d.ts

```diff
@@ -98,6 +98,26 @@ declare module "webhook-discord" {
      */
     public setTime(timestamp?: number): MessageBuilder;
 
+    /**
+     * Set the title for the discord hook
+     * @param title the webhooks´ title
+     */
+    public setTitle(title: string): MessageBuilder;
+
+    /**
+     * Set the author of the embedded object
+     * @param author the authors name
+     * @param iconURL the icon url
+     * @param url the url
+     */
```

Comment:
Should mirror the JavaScript:
```suggestion
    /**
     * Set the author of the embed
     * @param {string} author Name of the author
     * @param {string} iconURL Optional: Icon URL of the author
     * @param {string} url Optional: Link to the author
     */
```